### PR TITLE
Makefile - Remove dynamic file creation via perl.

### PR DESCRIPTION
**Pull request by jomey**
Just had a little surprise trying to compile this on our CHPC installation, where the perl version is different and does not create valid header and source files. Instead of trying to make the perl script work again, I decided to remove it since the dynamically created files are present and match the `.ini`. files.

Guess Danny was ahead of time, when introducing this early version of meta programming.

### CODE REVIEW

**Review by scotthavens — COMMENTED** [review]
What happens to the files that this creates?

We don't use `imath` in our day to day anymore but it would be good to understand if we still can.

**Comment by jomey** [comment]
The header files were broken.

Instead of
```C
#define TOKEN_EOI               0
```

it created
```C
#define                0
```

My perl version:
> This is perl 5, version 16, subversion 3 (v5.16.3) built for x86_64-linux-thread-multi

**Comment by scotthavens** [comment]
So it still creates the files and imath works as far as you can tell?

**Comment by jomey** [comment]
Yes.

As far as I understand, this is what happens:
The two removed makefile calls use:
* [cmdcodes.in](https://github.com/USDA-ARS-NWRC/ipw/blob/master/src/bin/pixcalc/imath/cmdcodes.in)
* [tokentypes.in](https://github.com/USDA-ARS-NWRC/ipw/blob/master/src/bin/pixcalc/imath/tokentypes.in)

To generate the each respective `cmdcodes.h` and `cmdcodes.c` (likewise with tokentypes) with the `makeCode.pl`. Both (header and source) of the generated files are checked into the repo though and you won't notice that they actually get generated. Only reason I noticed that, is because my perl version created the broken version of both header files.

So basically it is an unnecessary step at this point as both `.in` files are not changing.

You can try it yourself and go the `ipw/src/bin/pixcalc/imath/` dir, add an entry to each or both of the `.in` files and call make.

**Review by scotthavens — APPROVED** [review]
Thanks Joe, you are now an IPW master!
